src/lib/registration/addon.rb

```diff
@@ -338,6 +356,22 @@ def eula_acceptance_needed?
       !eula_url.to_s.strip.empty?
     end
 
+    STATUS_TRANSLATIONS = {
```

Comment:
The name is a bit confusing, these are actually not translations, `N_()` actually returns the original string. Maybe `STATUS_NAMES` or something like that would be more clear.